incubator/helm-release-pruner/values.yaml

```diff
@@ -9,9 +9,11 @@ job:
   backoffLimit: 3
   restartPolicy: Never
   schedule: "0 */4 * * *"
-  serviceAccountName: helm-release-pruner
   dryRun: true
   debug: false
+  serviceAccount:
+    create: true
+    # name: ExistingServiceAccountName
```

Comment:
Normally it's better to leave this uncommented so that you can pass a `helm lint --strict`. Just a note for future reference, non-blocking.